weave/flow/eval.py

```diff
@@ -119,9 +122,9 @@ def function_to_evaluate(question: str):
     ```
     """
 
-    dataset: Union[Dataset, list]
+    dataset: Union[Dataset, list[dict]]
```

Comment:
For future: These are common enough that it might be worth pulling out into a new type like `DatasetLike = Dataset | list[dict]`

Reply:
Fixed in next commit